README.md

```diff
@@ -375,6 +375,36 @@ wnbd-client.exe reset-opt LogLevel
 
 Passing the ``--persistent`` flag will remove the persistent setting as well.
 
+Limitations
+===========
+
+### CSV support
+
+At the moment, the Microsoft Failover Cluster can't use WNBD disks as
+Cluster Shared Volumes (CSVs) underlying storage. The main reason is that
+``WNBD`` doesn't support the *SCSI Persistent Reservations* feature yet.
```

Comment:
Has there been any work to implement this? Is there a tracked bug/feature I can follow?

Reply:
Not yet. I've just submitted this issue: https://tracker.ceph.com/issues/57109

To be honest, we haven't added any major features since our partnership with Suse ended. We've been mostly focusing on the CI, fixing bugs and increasing stability overall.

Reply:
@elipsion fyi, we've just submitted an experimental implementation: https://github.com/ceph/ceph/pull/48631